udplog/tap.py

```diff
@@ -55,6 +64,13 @@ def opt_redis_host(self, host):
         """
         self['redis-hosts'].add(host)
 
+    def opt_kafka_host(self, host):
+        """
+        Kafka broker connection string in the form <host>:<port>. Repeat for
+        selected few seed brokers from the cluster. The rest brokers will be
+        resolved automatically.
+        """
+        self['kafka-hosts'].add(host)
```

Comment:
In `__init__`, this is called `kafka-brokers`, which I like better, because each value is a host and a port. Also, as there are now two different names, this shouldn't work.